.github/workflows/swift-toolchain.yml

```diff
@@ -609,10 +609,37 @@ jobs:
           echo "PYTHON_LOCATION_amd64=$env:pythonLocation" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
           echo "PYTHON_LOCATION_arm64=${{ github.workspace }}\pythonarm64.${{ env.PYTHON_VERSION }}\tools" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
 
+      # Pin VS build tools to version 14.39.33519
+      # See https://github.com/actions/runner-images/blob/win22/20240514.3/images/windows/Windows2022-Readme.md
+      - name: Install VS Build Tools v14.39.33519
+        run: |
+          # Start-Process is required because setup.exe is non-blocking when run on GHA.
```

Comment:
Hmm sounds unlikely, but even if that's the case, you could do `& setup.exe --blah &2>1 | Out-String` to force it to wait and return a string

Reply:
Thanks for the tip! That works great: https://github.com/thebrowsercompany/swift-build/actions/runs/9911596581